### ISSUE TITLE
4D render

### ISSUE THREAD

**Issue by thangnd2000** [hustvl/4DGaussians#193]
Can I create a full range of rotations so I can have a quality output at all angles? for example 360 ​​degrees.

**Comment by weihan1** [comment]
the current code already does that with generateCameraFromTransforms in scene/dataset_readers.py

**Comment by thangnd2000** [comment]
Can I add motion to the point cloud file? Currently it cannot display animation.

**Comment by weihan1** [comment]
yes u just need have a bunch of cameras with time from 0 to 1 and then render the point clouds for each camera.

**Comment by thangnd2000** [comment]
Can I combine them into a single point cloud file?

**Comment by IQRAASGHAR1999** [comment]
@thangnd2000 have you get solution for this?
